### PR TITLE
Contrast 33138

### PR DESCRIPTION
**Pull request by rduran0**
Added article for Azure DevOps extension.

### CODE REVIEW

File: content/tools/extensions/AzureDevOpsExtension.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+* Complete all the fields with the required data. You can find the values for all the fields in the Contrast UI by going to the [**user menu > Your Account > Profile** page](user-account.html#profile).
+
+<a href="assets/images/AzureDevOps_service_connection.png" rel="lightbox" title="Service Connection fields"><img class="thumbnail" src="assets/images/AzureDevOps_service_connection.png"/></a>
+
+> **Note: ** Your **Contrast URL** should not include */Contrast* at the end; only the host is required.
+
+## Configuration for Release Gate
+
+### Step one
+
```
Comment (kaitlynmortimer) [inline content/tools/extensions/AzureDevOpsExtension.md]: @rduran0 Are there any images for this step?

File: content/tools/extensions/AzureDevOpsExtension.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+## Setup and Configuration
+
+### Step one
+
+* Follow [Microsoft's instructions](https://docs.microsoft.com/en-us/azure/devops/marketplace/install-extension?view=azure-devops-2019) to install the Contrast extension, **Contrast Integration**. 
+
+### Step two
+
+* Go to your **Project settings** at the bottom of you side bar.
```
Comment (kaitlynmortimer) [inline content/tools/extensions/AzureDevOpsExtension.md]: In what site/account does the user start on step two?

File: content/tools/extensions/AzureDevOpsExtension.md (hunk 1 above)
Comment (rduran0) [reply content/tools/extensions/AzureDevOpsExtension.md]: No. But I can get you some. And now that you mention it, that should make it less confusing for newcomers.

File: content/tools/extensions/AzureDevOpsExtension.md (hunk 2 above)
Comment (rduran0) [reply content/tools/extensions/AzureDevOpsExtension.md]: Azure DevOps project page.

**Pull request merged**